docs/api-docs/getting-started/rest-api-authentication.mdx

```diff
@@ -30,46 +32,37 @@ Most APIs that work with store API accounts use the `access_token` to authentica
+  Some resources are only accessible to the API account that created them. These include webhooks and metafields. If you need to revoke a store-level API account, plan accordingly.
 </Callout>
 
+## Account-level API accounts
 
-## App API accounts
+Merchants can generate account-level API credentials when they create an account API in their store's control panel (**Settings** > **API** > **Account API Tokens**). This type of API account grants merchants the ability to manage all stores tied to the specific account and not the user that created it. With account-level credentials, merchants can secure the OAuth scopes necessary to manage users on a larger scale.
 
-You can [create app API accounts](#obtaining-app-api-credentials) in the [Developer Portal](https://devtools.bigcommerce.com). Most apps use access tokens generated from the API account's `client_ID`, `client_secret`, and a grant `code` to read and change store data once the store owner installs and authorizes the app. [Generate access tokens](#app-access-tokens) with the BigCommerce-initiated grant code authorization flow.
+Since this API token is tied to a specific account, the token remains active unless the Merchant's account becomes inactive. The account-level API allows merchants to make API calls to our platform, with the help of our API Gateway, allowing merchants to provide an automated onboarding or offboarding experience on an enterprise level. There are other ways that BigCommerce's account-level API can assist with handling all provisioned users. For example, merchants can leverage an account-level token to perform actions like  generate records for compliance reporting, subscription management, and more. 
+For more information, see [Create an account-level API Account](https://support.bigcommerce.com/s/article/Store-API-Accounts?language=en_US#account-level).
 
-Some APIs use app API accounts to implement alternative authentication patterns. For a summary of all our authentication methods, see [Authentication and Example Requests](/api-docs/getting-started/authentication).
-
-For more on working with apps, see our [Guide to Building Apps](/api-docs/apps/guide/intro). The sections on [Implementing OAuth](/api-docs/apps/guide/auth) and [Callback Handlers](/api-docs/apps/guide/callbacks) are particularly relevant to generating access tokens.
-
-### Obtaining app API credentials
-
-To create an app and its associated API account, consult our article on [Managing Apps in the Developer Portal](/api-docs/apps/guide/developer-portal#create-an-app).
-
-### App access tokens
-
-App API accounts do not come pre-configured with an access token. Each time a store installs your app, BigCommerce initiates a grant code authorization flow to help your app generate a dedicated access token for that store. For further details, see [Authenticating an app](/api-docs/apps/guide/auth).
-
-### Revoking app API credentials
+>>>>Start: I think we should remove this part for now <<<
```

Comment:
@stanzikratelbc, If you search for >>>> you'll see some comments about revoking account-level api credentials. We don't have that supporting information yet. I suggest we remove it but I'll let you make that call. 👍